drivers/net/mlx5/mlx5_rxtx.c

```diff
@@ -2242,6 +2243,51 @@ rxq_cq_to_ol_flags(struct rxq *rxq, volatile struct mlx5_cqe *cqe)
 	return ol_flags;
 }
 
+#define MLX5_VPMD_RXQ_RPLNSH_THRESH  (64U)
+
+/**
+ * Replenish buffers for RX in bulk.
+ * Backported from DPDK 17.11
+ *
+ * @param rxq
+ *   Pointer to RX queue structure.
+ * @param rel_data_offset
+ *   Should be 0 for ETH, GRH_HDR_LEN for IPoIB
```

Comment:
need to explain what this value means, and not with what values it should be set.